Jenkinsfile

```diff
@@ -8,12 +8,12 @@ elifePipeline {
     node('containers-jenkins-plugin') {
         stage 'Build images', {
             checkout scm
-            sh './build.sh'
+            sh "IMAGE_TAG=${commit} ./build.sh"
```

Comment:
I was wondering whether it would be sensible to replace the `build.sh` and `run.sh` with `docker-compose build` an `... up`

Reply:
and `docker-compose down -v` in the `finally`. Yes, please refactor (now that every change will be built and tested in CI)